cmd/podman/containers/create.go

```diff
@@ -409,6 +409,8 @@ func createPodIfNecessary(cmd *cobra.Command, s *specgen.SpecGenerator, netOpts
 	infraOpts := entities.NewInfraContainerCreateOptions()
 	infraOpts.Net = netOpts
 	infraOpts.Quiet = true
+	infraOpts.ReadOnly = true
+	infraOpts.ReadOnlyTmpFS = false
```

Comment:
Could we set it to `true`?  We do it for the service-container which is essentially the same as an infra-container.

Reply:
The infra container sets it to `true` and I wonder why they are configured differently.

Reply:
Switched that one to false also.